### PR TITLE
Export unformatted ANSI escape codes

### PR DESCRIPTION
**Pull request by kevva**
For https://github.com/chalk/slice-ansi/issues/12. Needs some documentation, just wanted to get some input first :).

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -86,12 +87,20 @@ function assembleStyles() {
...
 		Object.defineProperty(styles, groupName, {
 			value: group,
 			enumerable: false
 		});
+
+		Object.defineProperty(styles, 'codes', {
+			value: codes,
+			enumerable: false,
+			configurable: true
+		});
```
Comment (kevva) [inline index.js]: Maybe just set `styles.codes = codes` directly?

Comment (Qix-) [inline index.js]: No this is fine; in fact, I'd remove the configurable entry.

Comment (kevva) [inline index.js]: Hm yeah. For some reason it `threw` yesterday without it, but worked now so must've done something else wrong.

**Comment by sindresorhus** [comment]
LGTM. Can you add some docs?

**Comment by kevva** [comment]
Yup, done.

File: readme.md [hunk 2]
```diff
@@ -102,6 +103,15 @@ By default, you get a map of styles, but the styles are also available as groups
 console.log(style.color.green.open);
 ```
 
+The raw ANSI escape codes used to create the styles is available under `style.codes` which returns a `Map` with the open codes as keys and close codes as values.
+
+###### Example
```
Comment (Qix-) [inline readme.md]: Is this many indents necessary?

File: readme.md [hunk 3]
```diff
@@ -102,6 +103,15 @@ By default, you get a map of styles, but the styles are also available as groups
 console.log(style.color.green.open);
 ```
 
+The raw ANSI escape codes used to create the styles is available under `style.codes` which returns a `Map` with the open codes as keys and close codes as values.
```
Comment (Qix-) [inline readme.md]: Should expand upon what "raw" means.

**Comment by Qix-** [comment]
Other than a super minor nit, it LGTM.

File: readme.md (hunk 3 above)
Comment (kevva) [inline readme.md]: Hm, I don't know if it's the correct wording tbh. Do you have any suggestions?

Comment (Qix-) [inline readme.md]: > Raw escape codes (i.e. without the CSI escape prefix `\x1b[`) are available under `style.codes`, which returns a `Map` with the open codes as keys and close codes as values.

Comment (kevva) [inline readme.md]: Thanks :).

**Pull request merged**